scripts/enrich.zeek

```diff
@@ -1,123 +1,123 @@
-@load ./extract_cve
+@load Corelight/Suricata
```

Comment:
if this requires Corelight-specific stuff, should it even be here?

Reply:
This will remain for now. :(